usaspending_api/recipient/models.py

```diff
@@ -120,11 +120,8 @@ class Meta:
         #         public.recipient_profile USING btree (recipient_name DESC NULLS LAST)
         #     SQL: CREATE INDEX recipient_profile_recipient_unique_id_d ON
         #         public.recipient_profile USING btree (recipient_unique_id DESC NULLS LAST)
```

Comment:
Can you update this comment above to also mention the new UEI indexes that are manually created with RunSQL?

Reply:
Done.